src/Mail/Mailable.php

```diff
@@ -47,6 +54,9 @@ public function buildViewData()
      */
     public function withSerializedData($data)
     {
+        // save current locale in $data to be serialized
+        $data['_saved_locale'] = App::getLocale());
```

Comment:
Key should probably be `_current_locale` instead, and I'd recommend something like the following:
```php
$defaultData = [
    '_current_locale' => App::getLocale(),
];

$data = array_merge($defaultData, $data);
```
to allow developers to override that value if desired.

Reply:
good idea.